fitlins/cli/run.py

```diff
@@ -155,7 +157,9 @@ def run_fitlins(argv=None):
     work_dir = mkdtemp() if opts.work_dir is None else opts.work_dir
 
     fitlins_wf = init_fitlins_wf(
-        opts.bids_dir, derivatives, deriv_dir, opts.space, model=model,
+        opts.bids_dir, derivatives, deriv_dir, opts.space,
+        desc=opts.desc,
```

Comment:
If you accept the above suggestion, this variable will change:

```suggestion
        desc=opts.desc_label,
```

Reply:
Ha, I have learned something new about argparse today! I didn't know that an argument with a dash in the name can be parsed with an underscore. :-)